### PR TITLE
build: login page

### PR DESCRIPTION
**Pull request by abdou059**

### CODE REVIEW

File: client/src/pages/Login/index.tsx [hunk 1]
```diff
@@ -1,5 +1,84 @@
...
+        </div>
+      </div>
+      <div className="formPage">
+        <form className="formLogin" onSubmit={formik.handleSubmit}>
+          <h3>LOG IN</h3>
+          <TextField
+            fullWidth
+            id="email"
+            name="email"
+            label="Email"
```
Comment (husamkamal) [inline client/src/pages/Login/index.tsx:53]: the admin login using username not email

File: client/src/pages/Login/index.tsx [hunk 2]
```diff
@@ -1,5 +1,84 @@
...
+    },
+  });
+
+  return (
+    <div className="loginPage">
+      <div
+        className="loginImg"
+      >
+        <div className="decsLogin">
+          <h3>LOG IN</h3>
```
Comment (LinaYahya) [inline client/src/pages/Login/index.tsx]:
```suggestion
```
Remove this line no need to have a login as a title, As we have title at the form section

File: client/src/pages/Login/index.tsx [hunk 3]
```diff
@@ -1,5 +1,84 @@
...
+            The better way
+            to
+            buy or sell a used car
+          </h2>
+          <p>Great Value | Trusted Quality | All Online</p>
+        </div>
+      </div>
+      <div className="formPage">
+        <form className="formLogin" onSubmit={formik.handleSubmit}>
+          <h3>LOG IN</h3>
```
Comment (LinaYahya) [inline client/src/pages/Login/index.tsx]:
```suggestion
       <h3>LOG IN</h3>
        <form className="formLogin" onSubmit={formik.handleSubmit}>
```
move h3 before form section

**Review by husamkamal — CHANGES_REQUESTED** [review]
It`s great work but I think you want to make the space between the filed more bigger see the Figma design

**Review by LinaYahya — CHANGES_REQUESTED** [review]
Good Job @abdou059 
Small changes to make

**Review by LinaYahya — COMMENTED** [review]
Maybe we can create a separate component for `LoginForm`, and then we can pass schema and initial values as props,
And then handle input based on initial values props

**Review by husamkamal — APPROVED** [review]

**Review by tariqabusada — APPROVED** [review]

**Review by mai-mohammed — APPROVED** [review]

**Review by LinaYahya — APPROVED** [review]

**Pull request merged**